Documentation/Blazorise.Docs/Pages/Blog/2024-07-19_HowToStyleBlazoriseComponents/Index.md

```diff
@@ -0,0 +1,162 @@
+﻿---
+title: How to style Blazorise components
+description: Discover
+permalink: /blog/how-to-style-blazorise-components
+canonical: /blog/how-to-style-blazorise-components
+image-url: /img/blog/2024-07-19/how-to-style-blazorise-components.png
+image-title: How to style Blazorise components
+author-name: Giorgi
+author-image: giorgi
+posted-on: Jul 19th, 2024
+read-time: 7 min
+---
+
+# Styling Blazorise components
+
+Hello! today we are going to go over how to style Blazorise components!
+
+As your know, Blazorise is a framework-agnostic library, this means we have a lot of options when it comes to choosing what type of framework we want to use.
+
+Check out the [quick start](https://blazorise.com/docs/start) guide here to get started with Blazorise.
+
+## In this article, we will go over the following:
+
+- Styling Blazorise components#How does CSS work with Blazorise?|How does CSS work with Blazorise?
```

Comment:
I'm not sure what's going on here? Is this supposed to display like this? Doesn't read well to me? Maybe these should be anchors?
![image](https://github.com/user-attachments/assets/81edc8d7-d58f-49cf-a62f-e02007e2c947)

Reply:
They should indeed be links indeed, I will fix those